### ISSUE TITLE
Aedes

### ISSUE THREAD

**Issue by shortyishere** [zuhito/node-red-contrib-mqtt-broker#28]
Wondering if you should switch to aedes...its being maintained and faster..
https://www.npmjs.com/package/aedes

**Comment by martin-doyle** [comment]
Thank you for the proposal. I will check the other MQTT broker.

**Comment by martin-doyle** [comment]
It seems to work pretty good. I would suggest creating a new Node Red node because this is a major change. What do you think?

**Comment by qwertangel** [comment]
yeah. better to create new node. that will save tons of effort in making node backwards compatible across mosca and aedes.

**Comment by martin-doyle** [comment]
👍

**Comment by qwertangel** [comment]
btw mosca is causing trouble with nodejs 12 since mosca is using old  dependencies which breaks the installation (building of node-gyp fails). so will really appreciate if devs can create a aedes node-red node  .

**Comment by martin-doyle** [comment]
Please check the new [node-red-contrib-aedes](https://github.com/martin-doyle/node-red-contrib-aedes).

**Comment by qwertangel** [comment]
wow.. that was awesome @martin-doyle !

**Comment by qwertangel** [comment]
just replaced my mosca with aedes node. So far so good. Great work martin !!!

**Comment by mbc99** [comment]
I also replaced the mosca node for the aedes node and the transition was totally smooth. No problems so far!!

Great job!!
